### DISCUSSION TITLE
比利cos正片

### DISCUSSION THREAD

**Discussion by zhangrencos** [share121/inter-knot#3236] [其他]
发在米游社没人看，不知道这里有没有人看，明明我这套拍得质量很高啊。
绝区零伦敦线下活动
![1](https://github.com/user-attachments/assets/2f9e5afb-c0f2-4f74-801b-e1121abde304)
![2](https://github.com/user-attachments/assets/ab82cbc3-465b-4400-bbb9-6ffe14b61d4f)
![3](https://github.com/user-attachments/assets/6c66108d-16f0-4f4d-be37-f7ba41c6f398)

**Comment by OneMiaomiao** [comment]
很厉害！ 真的很还原！ 辛苦了！ 喜欢！

**Comment by ZhuYuan-148** [comment]
很帅，加油哦

**Comment by asdfghzxc12** [comment]
豪堪！

**Comment by LT-UT** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/03/21/fcba628ce15730cf0dbf480ec338f646_3898608417719823301.png" alt="丽都漫步-比利" width="75" >非常好看

**Comment by Guessme404** [comment]
好看，绳网要是能点赞就好了<img src="https://img-static.mihoyo.com/communityweb/upload/408018316100a8a8a846c68acedaa191.gif" alt="米游姬 卖萌" width="75" >

**Reply by 6078HITHEDAY** [reply]
可以的

**Comment by FFlaming0** [comment]
蟀！

**Comment by yzysrewrwqr** [comment]
<img src="https://bbs-static.miyoushe.com/static/2023/11/16/810dd11a4d16759f16ca078596aab6b1_422007669816866994.png" alt="哲-让我想想" width="75" >
加点比利的枪击特效，色调按游戏里调的低饱和些

**Comment by Xmug** [comment]
![无相之火觉得很赞](https://github.com/user-attachments/assets/02beb265-5997-48e8-be65-8b9f7284b28e)

**Comment by zzyuJPG** [comment]
帅<img src="https://bbs-static.miyoushe.com/static/2024/03/21/fcba628ce15730cf0dbf480ec338f646_3898608417719823301.png" alt="丽都漫步-比利" width="75" >

**Comment by trifortune** [comment]
帅的<img src="https://upload-bbs.mihoyo.com/upload/2022/11/14/62158b26331a045bbaabf48d1fc5c8eb_980401343708652819.png" alt="阿姬灵感" width="75" >

**Comment by Nie7bai** [comment]
帅呢<img src="https://bbs-static.miyoushe.com/static/2024/03/21/fcba628ce15730cf0dbf480ec338f646_3898608417719823301.png" alt="丽都漫步-比利" width="75" >

**Comment by RaphaelHider** [comment]
帅的<img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" ><img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" ><img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" >

**Comment by CC9704-star** [comment]
厉害。

**Comment by SKWTEA** [comment]
非常好的比利，使我的机油旋转

**Comment by Kamisato100** [comment]
不错哦，挺帅的

米游社的推送机制是这样抽象的，辛辛苦苦整了个活可能就只有几个观看压根没评论和点赞，随便水一水反而几百观看，只能说烂透了。
同样的内容发P站保底千把观看几百赞<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by ATEFSTARS** [comment]
帅帅帅😍

**Comment by Icather** [comment]
帅帅帅帅帅

**Comment by wjw-0629** [comment]
帅的嘞

**Comment by afishwith** [comment]
cool

**Comment by sicusa** [comment]
帅帅帅<img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" >

**Comment by JellyfishFly** [comment]
帅<img src="https://bbs-static.miyoushe.com/static/2023/11/16/e03be25a7b2bc86f2b1909dddb6ca915_6686505940047795068.png" alt="安东-元气满满" width="75" >

**Comment by jiangshixie** [comment]
老师拍得好帅！<img src="https://bbs-static.miyoushe.com/static/2024/07/31/fdc5e0e30e577fb208f29effa93510ed_8442612635207222552.png" alt="云璃 期待" width="75" >

**Comment by YolandaWang-0** [comment]
攒呦<img src="https://bbs-static.miyoushe.com/static/2024/03/21/fcba628ce15730cf0dbf480ec338f646_3898608417719823301.png" alt="丽都漫步-比利" width="75" >

**Comment by jidianzhuxing** [comment]
好看，很帅。<img src="https://upload-bbs.miyoushe.com/upload/2023/01/17/e5cf284f0136dd3dd6d161bc100690ad_2530421789229081234.png" alt="点赞呜呜" width="75" >

**Comment by qinglanfeng** [comment]
帅帅帅！

**Comment by The-Place-Where-Winter-Sieeps** [comment]
帅！比利 帅！
<img src="https://img-static.mihoyo.com/communityweb/upload/d2ebde3faa8c113f5c54b171d907f271.png" alt="比利 摇摆" width="75" >

**Comment by zytcw** [comment]
好康

**Comment by The-Debris-Of-The-Past-Ttars** [comment]
好看爱看喜欢看

**Comment by ID104772508** [comment]
厉害呀

**Comment by ltsraining** [comment]
好帅！老师穿的什么束胸可不可以分享下，看起来效果好棒！

**Comment by Robin-txdy** [comment]
老师辛苦了！

**Comment by tuneinnocent** [comment]
好帅呀～面部表情是屏幕显示的吗？

**Comment by nzj2004** [comment]
帅帅帅

**Comment by JiuXiat** [comment]
Cool~~!

**Comment by FaeCircuit** [comment]
cool~

**Comment by Andrea0126** [comment]
超帥的！！！
![img-1725796459345db277c8dfab32597244128f10c33abc20b83a3e03651e9ecb7ccc47a4ee7db3d](https://github.com/user-attachments/assets/fd1781c2-6de8-4f98-bdc3-22ab0095dff8)

**Comment by Junjun280** [comment]
我去，好帅！！

**Comment by kunga555** [comment]
泰裤辣

**Comment by 1234ADS-WQ** [comment]
确实6<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by HLXS-HLXS** [comment]
太酷啦

**Comment by nabeilishi** [comment]
帅
